### DISCUSSION TITLE
Add more exercises

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#225] [Ideas]
Let's collect exercises here. Please include:
* name
* muscles worked (target and synergist)
* image from gymvisual.com
* instructions how to perform it
* ideally a youtube video link showing it

**Comment by yrinkel** [comment]
**Name:** Captain Chair Straight Leg Raises, Bodyweight

**Type:** Core
**Target:** Rectus Abdominis, Obliques
**Synergist:** Quadriceps, Gluteus Maximus, Gluteus Medius
**Image:** 
https://gymvisual.com/12286-large_default/captains-chair-straight-leg-raise.jpg
**YouTube video:** https://images.app.goo.gl/1ykFAMPNLPPBeeCb9
**Instructions:**
1. Place your back against the back pad on a captain’s chair station, grab the handles, and rest your forearms on the arm pads. 
2. Support your body weight on your arms and let your feet hang straight down. 
3. With a slight bend in your knees, flex your abs and lift your legs up in an arc until your thighs are parallel to the ground. 
4. Reverse the movement and return to the starting position.

**Comment by yrinkel** [comment]
**Name:** Rear Delt Row, Barbell

**Type:** Pull, Upper
**Target:** Deltoid Posterior
**Synergist:** Trapezius Lower Fibers, Trapezius Middle Fibers, Trapezius Upper Fibers, Infraspinatus, Teres Major, Teres Minor
**Image:** https://gymvisual.com/19315-large_default/barbell-rear-delt-row.jpg
**YouTube video:** https://youtu.be/XagV4XfG9fo?si=kw00pTNIW2XztOFb
**Instructions:** Similar to a Barbell Row but the bar comes higher (mid-chest) to target the rear deltoids. 

1. Set Up

Load a barbell on the floor in front of you, bend over at the waist, and grab the bar with an overhand (palms facing you) grip a little wider than shoulder-width apart. Position your feet about shoulder-width apart, flatten your back so that it’s parallel to the floor, and let your arms hang straight down.

2. Row

Pull the bar straight up until it touches your mid-chest, keeping your spine straight and back parallel to the floor. Allow your elbows to flare at about a 60-degree angle relative to your torso.

3. Descend

Reverse the movement and return to the starting position. This is a mirror image of what you did during the row.

Don’t let the barbell fall back to the starting position or try to lower it slowly—the entire descent should be controlled but only take about a second.

**Comment by yrinkel** [comment]
**Name:** Rear Delt Row, Dumbbell

**Type:** Pull, Upper
**Target:** Deltoid Posterior
**Synergist:** Trapezius Lower Fibers, Trapezius Middle Fibers, Trapezius Upper Fibers, Infraspinatus, Teres Major, Teres Minor
**Image:** https://gymvisual.com/1150-large_default/dumbbell-rear-delt-rowshoulder.jpg

https://i.pinimg.com/originals/09/0d/05/090d05fe03ef2509910d669af3d69a51.gif

**YouTube video:** https://youtu.be/VQ3Ji5hD-VI?si=LMhFVadjoRfvzcd4
**Instructions:** Similar to a Dumbbell Row but the dumbbells comes higher (mid-chest) to target the rear deltoids. 

For the dumbbell rear delt row, you use [dumbbells](https://legionathletics.com/dumbbell-workouts/) instead of a barbell. This is beneficial because it allows you to [train](https://pubmed.ncbi.nlm.nih.gov/23604798/) through a slightly longer [range of motion](https://legionathletics.com/full-or-partial-reps/), which is generally better for muscle and strength gain. It also trains each side of your body independently, which helps you identify and correct [muscle and strength imbalances](https://legionathletics.com/muscle-imbalances/).

That said, the dumbbell rear delt row requires more coordination and balance than the barbell version, so it might not be as suitable for beginners. Most people also find they can’t lift quite as much weight using dumbbells as a barbell, which is typically worse for muscle growth.

**Comment by yrinkel** [comment]
**Name:** Triceps Kickback, Dumbbell

**Type:** Push, Upper
**Target:** Triceps Brachii, Brachialis
**Synergist:** Deltoid Posterior
**Image:** https://gymvisual.com/1019-large_default/dumbbell-kickback.jpg

**YouTube video:** https://youtu.be/6SS6K3lAwZ8
**Instructions:**
1 - Hold a dumbbell in your right hand. 
2 - Plant your left knee and arm firmly on a bench and your right foot on the floor a foot or two from the bench. Position your right elbow slightly behind your torso, but bend it at about 90 degrees so the weight hangs toward the floor. 
3 - Push the weight toward your hip by straightening your elbow.
4 - Reverse the movement and return to the starting position

**Comment by yrinkel** [comment]
**Name:** Reverse-Grip Bench Press, Barbell

**Type:** Push, Upper
**Target:** Pectoralis Major Clavicular Head
**Synergist:** Triceps Brachii, Pectoralis Major Sternal Head, Deltoid Anterior
**Image:** 
https://gymvisual.com/23891-large_default/barbell-reverse-grip-bench-press.jpg
**Youtube Video:** https://youtu.be/liJ7o1x6TPg?si=LmXLHOEDIyYzVaO7
**Instructions:**
Start with the same setup than a normal bench press. 
To do the reverse-grip bench press, flip your grip around on the bar (so your palms face you). Grip the bar so that it crosses your palm diagonally, from the base of your index finger to the opposite edge of your wrist.

**Comment by yrinkel** [comment]
**Name:** Seated Triceps Extension, Dumbbell

**Type:** Push, Upper
**Target:** Triceps Brachii
**Synergist:** Pectoralis Major Sternal Head, Latissimus Dorsi, Teres Major
**Image:** https://gymvisual.com/8827-large_default/dumbbell-seated-triceps-extension.jpg

**Youtube Video:** https://legionathletics.com/wp-content/uploads/2022/08/450x338-half.gif

**Instructions:**

1. Set up

Sit up straight on a flat bench or with your back pressed against an upright bench with a low backrest. Grip one end of a dumbbell using both hands and lift it overhead so that your arms are straight. Maneuver your palms so that they’re flat against the end of the dumbbell and facing toward the ceiling. 

2. Descend

While keeping your upper arms as close to perpendicular to the floor as possible, lower the dumbbell behind your head by bending your elbows. Go as far as your flexibility allows or until you feel a deep stretch in your triceps.

3. Extend

Push the dumbbell toward the ceiling by straightening your elbows and return to the starting position. This is a mirror image of what you did during the descent.

Try to minimize your upper-arm movement as you lift the weight, as shifting your arms increases the chances of knocking your noggin with the dumbbell.

**Comment by yrinkel** [comment]
**Name:** Weighted Crunch, Dumbbell

**Type:** Core
**Target:** Rectus Abdominis
**Synergist:** Obliques, Erector Spinae, Iliopsoas
**Image:** https://gymvisual.com/19395-large_default/weighted-crunch-version-2.jpg


**Youtube Video:** https://youtu.be/kZvSaq192cg?si=uGuMwhrBSrBXdJSk

**Instructions:**
1. Lie on the floor with your feet wedged below heavy dumbbells or on a bench with your feet wedged under the foot pads. 
2. Using both hands, hold the weight against your chest with your arms crossed. 
3. Keeping your back flat, use your ab muscles to lift your chest toward the ceiling until your lower back comes off the bench or floor.
4. Reverse the movement and return to the starting position.

**Comment by PTVuser** [comment]
We would appreciate it if it could be included into the database, it is a very popular kind of STD exercise, all the info requested:
 
### ***Name: Straight Arm Pulldown, Cable**
[Type: Strength, Cable, Compound, Pull (Bilateral)]


***Muscles worked (target and synergist)**
Target Muscle Group: Lats
Synergist/Secondary Muscles: Abs, Biceps, Shoulders, Upper Back



***Instructions how to perform it**
(1) You will start by grabbing the wide bar from the top pulley of a pulldown machine and using a wider than shoulder-width pronated (palms down) grip. Step backwards two feet or so.
(2) Bend your torso forward at the waist by around 30-degrees with your arms fully extended in front of you and a slight bend at the elbows. If your arms are not fully extended then you need to step a bit more backwards until they are. Once your arms are fully extended and your torso is slightly bent at the waist, tighten the lats and then you are ready to begin.
(3) While keeping the arms straight, pull the bar down by contracting the lats until your hands are next to the side of the thighs. Breathe out as you perform this step.
(4) While keeping the arms straight, go back to the starting position while breathing in.
(5) Repeat for the recommended amount of repetitions.

Variations: Can also be performed with a shorter shoulder-width straight bar or with a rope attachment.



***Images from gymvisual.com**
Image: https://gymvisual.com/videos/2894-cable-straight-arm-pulldown.html
Animated GIF: https://gymvisual.com/animated-gifs/1731-cable-straight-arm-pulldown.html



***Youtube video links showing it:**
Video 1: https://www.youtube.com/watch?v=wcVDItawocI
Video 2: https://www.youtube.com/watch?v=G9uNaXGTJ4

**Comment by PTVuser** [comment]
We would also appreciate it if it could be included into the database, it is a very popular and useful exercise as well, all the info requested:
 
### ***Name: Standing Lat Pulldown, Cable**


***Muscles worked (target and synergist)**
Target Muscle Group: Lats
Synergist/Secondary Muscles: Biceps, Forearms, Shoulders, Upper Back, Core Muscles


***Instructions how to perform it**
SETUP: 
- Assume a standing position facing the cable machine.  
- Extend your arms upwards to grab the bar with your hands roughly shoulder width apart and your palms facing away from you. 
- Step away from the bench. Place one foot on the bench for stability and lean back slightly. 


ACTION:
1) Brace your core, bring your shoulder blades down and back, and pull the bar down until it touches the top of your rib cage. 
2) Pause briefly at the bottom, squeeze your lats, and slowly return to the starting position.
3) Maintain tightness in your core and repeat!

VERSIONS:
The line of pull of the standing pulldown falls somewhere in the middle between a pulldown and a row. Therefore, it targets different muscle fibers than can be stressed on the pulldown or row.
- Using a wide grip puts more of the stress on the upper lat muscle fibers.
- Switching to a narrow and neutral grip moves the focus down more toward the lower lats, but it still hits the lats differently than a standard cable row.
- Make sure to use a handle that"s wide enough to pass your rib cage for greater range of motion and stimulation of the lat muscle fibers.

***Image from gymvisual.com**
https://gymvisual.com/videos/15881-cable-standing-lat-pulldown.html

***Youtube video links showing it:**
Long Video: https://www.youtube.com/watch?v=HynaSdVSKMw
Short Video: https://www.youtube.com/watch?v=tcGqUzH2xxk

**Comment by PTVuser** [comment]
> Let's collect exercises here. Please include:
> 
> * name
> * muscles worked (target and synergist)
> * image from gymvisual.com
> * instructions how to perform it
> * ideally a youtube video link showing it

Is there a regular exercise database update cycle? Like once every month or quarterly? Please let us know. If communicated, people would know when to submit missing exercises. Thank you.

**Comment by legalmentecristian** [comment]
**Name:** Forearm Push Up, bodyweight

**Type:** Push, Upper
**Target:** Triceps Brachii
**Synergist:** Pectoralis Major Sternal Head, Pectoralis Major Clavicular Head, Anterior Deltoid, Coracobrachialis
**Image:** https://gymvisual.com/videos/5122-push-up-on-forearms.html#/47-style_type-basic_grey
**YouTube video:** https://www.youtube.com/watch?v=IqLInqqLNNE
**Instructions:**

_Preparation_

- Begin in a high plank position with your hands shoulder-width apart, positioned in line with your face.
- Keep your elbows slightly bent and pointed forward, maintaining a straight body line from head to heels.

_Execution_

- Lower your body by bending your elbows, allowing your forearms to move toward the ground—think of it like an inverted skullcrusher.
- Continue lowering until your head moves toward your hands and your forearms are nearly touching the floor.
- Press through your palms to extend your arms and return to the starting position.
- Maintain a tight core and straight body throughout the movement.
- Repeat for the desired number of reps.

**Comment by legalmentecristian** [comment]
**Name:** Inverted Bicep Curls, bodyweight

**Type:** Pull, Upper
**Target:** Biceps Brachii
**Synergist:** Trapezius Middle Fibers, Trapezius Lower Fibers, Rhomboids, Latissimus Dorsi, Teres Major, Posterior Deltoid, Infraspinatus, Teres Minor, Brachialis, Brachioradialis, Pectoralis Major Sternal Head
**Image:** https://gymvisual.com/videos/14490-bar-biceps-curl.html#/47-style_type-basic_grey
**YouTube video:** https://www.youtube.com/watch?v=JOzdwkeGC-A
**Instructions:**

_Preparation_

- Lie on your back under a fixed horizontal bar or suspension trainer (such as a smith machine or TRX).
- Grasp the bar with an underhand (supinated) grip, slightly wider than shoulder-width.
- Position your body in a straight line with heels on the ground and arms fully extended.

_Execution_

- Engage your biceps and pull your face toward the bar, keeping your elbows close to your body.
- Continue pulling until your head moves toward your hands, squeeze at the top.
- Lower yourself in a controlled manner to extend your arms and return to the starting position.
- Maintain a tight core and a straight body throughout the movement.
- Repeat for the desired number of reps.

**Comment by legalmentecristian** [comment]
**Name:** Reverse Nordic Curls, bodyweight

**Type:** Legs, Lower
**Target:** Rectus Femoris, Vastus Medialis, Vastus Lateralis, Vastus Intermedius
**Synergist:** Tensor Fasciae Latae
**Image:** https://gymvisual.com/videos/14324-bodyweight-kneeling-sissy-squat.html#/47-style_type-basic_grey
**YouTube video:** https://www.youtube.com/watch?v=c1sjDhdIqXk
Instructions:

_Preparation_

- kneel on a padded surface such as a Pilates mat or foam pillow to provide cushioning for your knees.

_Execution_

- Begin the movement by tensing your abs and glutes to brace your core while your torso is in a vertical position.
- While keeping your hips extended and torso rigid, gradually lean backwards by flexing at the knees.
- Descend as far as you can control the movement.
- Then return to the starting position by extending your knees.
- Repeat for the desired number of reps.

**Comment by legalmentecristian** [comment]
**Name:** Nordic Curls, bodyweight

**Type:** Legs, Lower
**Target:** Biceps Femoris, Semimembranosus, Semitendinosus
**Synergist:** Gastrocnemius, Sartorius, Gracilis, Popliteus, Pectoralis Major Sternal Head, Pectoralis Major Clavicular Head, Anterior Deltoid, Triceps Brachii, Coracobrachialis, Wrist Flexors
**Image:** https://gymvisual.com/videos/16055-inverse-leg-curl-with-bench-pads.html#/47-style_type-basic_grey
**YouTube video:** https://www.youtube.com/watch?v=_e9vFU9-tkc
**Instructions:**

_Preparation_

- Anchor your feet (partner holding feet down, door anchor, squat rack anchor, barbel with plates, etc) and kneel on a padded surface such as a Pilates mat or foam pillow to provide cushioning for your knees.

_Execution_

- Begin the movement by tensing your abs and glutes to brace your core while your torso is in a vertical position.
- Attempt to keep a straight line between your knees, hips, and shoulders as you slowly lower to the floor or object in front of you.
- Descend as far as you can control the movement.
- Then catch yourself with your hands and push back up while flexing your knees to return to the starting position.
- Repeat for the desired number of reps.

**Comment by PTVuser** [comment]
Behind The Neck for Press is already included in the Liftosaur, but Behind The Neck Lat Pulldown is also a very popular exercise as well, please find below all the info requested for missing exercises:
	 
### ***NAME: (Wide Grip) Behind Neck Pulldown, Cable**
[Type: Strength, Compound, Pull (Bilateral)]

	
***Muscles worked (target and synergist)**
Target Muscle Group: Lats
Synergist/Secondary Muscles: Biceps, Shoulders, Upper Back, Abs


***Instructions how to perform it**
1. Attach a wide grip handle to the lat pulldown machine and assume a seated position.
2. Grasp the handle with a pronated grip (double overhand) and initiate the movement by depressing the shoulder blades and then flexing the elbow while extending the shoulder.
3. Pull the handle behind your neck until it nearly touches at the base of your traps and then slowly lower the handle back to the starting position under control.
4. Repeat for the desired number of repetitions.	

TIPS
- Keep some tone through your abdominals as you pull the bar into your body to ensure you don’t arch excessively through the spine.
- Don’t allow momentum to dictate the movement, control the dumbbells throughout the entirety of each rep.
- If you feel your biceps being overused and your back remaining under active, consider utilizing a false grip (i.e. don’t wrap the thumb around the dumbbell).
- Don’t allow the head to jut forward as you pull.
- Similarly, ensure the shoulder blade moves on the rib cage. Don’t lock the shoulder blade down and just move through the glenohumeral joint.
- Allow the shoulder to internally rotate and shrug slightly at the top of the movement. You will obviously reverse the movement and depress the shoulder blade before you pull with the arm.


***Image from gymvisual.com**
https://gymvisual.com/animated-gifs/3097-cable-wide-grip-behind-neck-pulldown.html

***Youtube video links showing it:**
https://www.youtube.com/watch?v=NOZ37M90bSo

**Comment by PTVuser** [comment]
Also very popular and useful exercise, all the info requested:
	 
### ***Name: Reverse Grip (Underhand) Lat Pulldown, Cable**
[Type: Strength, Compound, Pull (Bilateral)]

***Muscles worked (target and synergist)**
Target Muscle Group: Lats
Synergist/Secondary Muscles: Biceps, Shoulders, Upper Back, Abs
	
	
***Instructions how to perform it**
1. Attach a wide grip handle to the lat pulldown machine and assume a seated position.
2. Grasp the handle with a supinated grip (double underhand) just inside of shoulder width.
3. Initiate the movement by depressing the shoulder blades and then flexing the elbow while extending the shoulder.
4. Pull the handle towards your body until the elbows are in line with your torso and then slowly lower the handle back to the starting position under control.
5. Repeat for the desired number of repetitions.

TIPS

- Keep some tone through your abdominals as you pull the bar into your body to ensure you don’t arch excessively through the spine.
- Don’t allow momentum to dictate the movement, control the weight throughout the entirety of each rep.
- If you feel your biceps being overused and your back remaining under active, consider utilizing a false grip (i.e. don’t wrap the thumb around the dumbbell).
- Don’t allow the head to jut forward as you pull.
- Similarly, ensure the shoulder blade moves on the rib cage. Don’t lock the shoulder blade down and just move through the glenohumeral joint.
- Allow the shoulder to internally rotate and shrug slightly at the top of the movement. You will obviously reverse the movement and depress the shoulder blade before you pull with the arm.	


***Image from gymvisual.com**
https://gymvisual.com/videos/5662-lever-reverse-grip-lateral-pulldown-plate-loaded.html

***Youtube video links showing it:**
https://www.youtube.com/watch?v=8DAFwftgZHQ
